clients/js/test/digitalAsset.test.ts

```diff
@@ -254,3 +321,24 @@ function createDigitalAssetWithSecondCreator(
     ]),
   });
 }
+
+async function createDigitalAssetWithVerifiedCollection(
+  context: Parameters<typeof createDigitalAssetWithToken>[0],
+  input: Parameters<typeof createDigitalAssetWithToken>[1] & {
```

Comment:
Can we 'omit' the verified field here so we don't need to pass `false` all the time?